netci.groovy

```diff
@@ -633,6 +639,7 @@ combinedScenarios.each { scenario ->
                             }
                             break
                         case 'Ubuntu':
+                        case 'Ubuntu15.10':
```

Comment:
Nit: should we update existing usage of 'Ubuntu' to be 'Ubuntu14.04` or something like that?